### DISCUSSION TITLE
关于webrtc中的AAC/H265

### DISCUSSION THREAD

**Discussion by 0xLevin** [yapingcat/gomedia#110] [Q&A]
你好，上次经过您的耐心指导，已经成功将264推到pion的webrtc实现了播放，感谢您的支持
现在又有两个问题想您请教。
已知默认情况下 webrtc 不支持aac与h265, 请问纯golang可以做到 aac->opus, h265->h264转换吗?

**Comment by yapingcat** [comment]
> 你好，上次经过您的耐心指导，已经成功将264推到pion的webrtc实现了播放，感谢您的支持 现在又有两个问题想您请教。 已知默认情况下 webrtc 不支持aac与h265, 请问纯golang可以做到 aac->opus, h265->h264转换吗?

不能
